mllib/src/main/scala/org/apache/spark/mllib/regression/GeneralizedLinearAlgorithm.scala

```diff
@@ -142,6 +147,11 @@ abstract class GeneralizedLinearAlgorithm[M <: GeneralizedLinearModel]
   protected def createModel(weights: Vector, intercept: Double): M
 
   /**
+   * Get if the algorithm uses addIntercept
+   */
+  def isAddIntercept: Boolean = this.addIntercept
```

Comment:
Heh, `should` is more natural although I'm used to `is` or `get` for getters in Java. I had the same reaction but think `is`is OK. It's more obviously a getter to my eyes.